core/arch/arm/kernel/entry_a32.S

```diff
@@ -516,6 +516,14 @@ shadow_stack_access_ok:
 	bl	console_init
 #endif
 
+#ifdef CFG_VIRTUALIZATION
+	/*
+	 * Reinitialize partition tables for each partition to
```

Comment:
I'm not sure if you can call it "reinitialization", because this is the first and only one place where are you initializing default partitions.

Reply:
I will change it to Initialize.